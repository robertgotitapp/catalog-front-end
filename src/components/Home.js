import React, { Component } from 'react';
import { CardGroup, Card } from 'react-bootstrap';
import { connect } from 'react-redux';
import Col from 'react-bootstrap/Col';
import CategoryList from './CategoryList';
import ItemList from './ItemList';
import { getItems } from '../actions/items';

class Home extends Component {
  render() {
    return (
      <CardGroup>
        <Col sm={3}>
          <CategoryList />
        </Col>
        <Col sm={9}>
          <ItemList />
        </Col>
      </CardGroup>
    );
  }
}

const mapDispatchToProps = {
  getItems,
};

export default connect(null, mapDispatchToProps)(Home);
