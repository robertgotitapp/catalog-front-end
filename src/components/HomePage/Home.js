import React, { Component } from 'react';
import { CardGroup } from 'react-bootstrap';
import { connect } from 'react-redux';
import Col from 'react-bootstrap/Col';
import CategoryList from './CategoryList';
import ItemList from './ItemList';

export class Home extends Component {
  render() {
    return (
      <CardGroup>
        <Col sm={3}>
          <CategoryList />
        </Col>
        <Col sm={9}>
          <ItemList />
        </Col>
      </CardGroup>
    );
  }
}

export default connect()(Home);
