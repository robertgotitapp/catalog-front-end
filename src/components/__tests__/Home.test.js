import React from 'react';
import { shallow } from 'enzyme';
import { Home } from '../Home';

describe('components/Home', () => {
  const container = shallow(<Home />);

  it('should render correctly', () => {
    expect(container).toMatchSnapshot();
  });
});
