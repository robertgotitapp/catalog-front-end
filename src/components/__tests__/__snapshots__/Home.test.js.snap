// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Home should render correctly 1`] = `
<CardGroup>
  <Col
    sm={3}
  >
    <Connect(CategoryList) />
  </Col>
  <Col
    sm={9}
  >
    <Connect(ItemList) />
  </Col>
</CardGroup>
`;
